### DISCUSSION TITLE
Mouse freeze and keyboard input latency

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#28401] [Q&A]
### Type

Performance

### Bug description

Sometimes the cursor (mouse only) freezes. It stops suddenly for about 500ms, then starts again. The keyboard is also interpenetrated later, with I'd say the same latency as for the mouse cursor. This also happened to a friend, which leads me to believe that it's a game bug. It happens on both Windows and Linux.

### Version

2024.521.2

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions?discussions_q=is%3Aopen+mouse+freeze

likely the same thing as one of these (many share a common cause)
